package.json

```diff
@@ -5,19 +5,16 @@
   "license": "MIT",
   "dependencies": {
     "@kiwicom/eslint-config": "^3.1.0",
-    "@kiwicom/fetch": "^2.2.0",
+    "@kiwicom/fetch": "^2.2.1",
     "@kiwicom/orbit-components": "^0.33.0",
-    "@kiwicom/relay": "^0.25.0",
+    "@kiwicom/relay": "^0.27.0",
     "babel-eslint": "^10.0.1",
     "eslint": "^5.14.1",
     "flow-bin": "^0.93.0",
     "graphql": "^14.1.1",
     "next": "^8.0.3",
     "react": "^16.8.3",
     "react-dom": "^16.8.3",
-    "react-relay": "^3.0.0",
-    "react-relay-network-modern": "^2.5.1",
-    "relay": "^0.8.0-1",
```

Comment:
What is this?! 😬

Reply:
Copy- pasted 🙈

Reply:
@tbergq In this case, fix the copy-paste source because `relay` is some unknown and 6 years old dependency: https://www.npmjs.com/package/relay 🙃